schema.graphql

```diff
@@ -839,8 +848,11 @@ type Dependency @entity {
 type DependencyRegistry @entity {
   "Unique identifier made up of dependency registry contract address"
   id: Bytes!
-  "Core contracts that this registry can provide dependeny overrides for"
-  supportedCoreContracts: [Contract!]! @derivedFrom(field: "dependencyRegistry")
+  "Core registry that defines the base set of contracts supported by this registry"
+  coreRegistry: CoreRegistry
+  "List of core contracts that are explicitly allowlisted on this registry, extending those on the CoreRegistry"
+  supportedCoreContractsOverride: [Contract!]!
+    @derivedFrom(field: "latestDepenencyRegistryOverrideAllowlistedOn")
```

Comment:
```suggestion
    @derivedFrom(field: "latestDependencyRegistryOverrideAllowlistedOn")
```

Reply:
fixed in [a981a0a](https://github.com/ArtBlocks/artblocks-subgraph/pull/328/commits/a981a0a44af5112297a3b25e5e34a708b237c261)